src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -181,6 +184,33 @@ def register_options(cls, register):
       ]
     )
 
+    cls.register_jvm_tool(
+      register,
+      'scalac-outliner',
+      classpath=[
+        JarDependency(
+          org='org.scala-lang',
+          name='scala-compiler',
+          rev='2.12.10',
+        ),
+      ]
+    )
```

Comment:
Yeah we should still shade it for the same reason as shading zinc: https://github.com/pantsbuild/pants/blob/5b99c2bbc6b8f870ed9ad239ce9477ee7ab7a2b5/src/python/pants/backend/jvm/subsystems/zinc.py#L68-L82, except the nailgun part. 

If it is ideal to keep this exactly the same as scala version, then it might be good to have `ScalaPlatform` handle with version to grab.